packages/deploy/deploy/400_asset/402_deploy_asset.ts

```diff
@@ -35,6 +37,7 @@ export default func;
 
 func.tags = ['Asset', 'Asset_deploy', 'L2'];
 func.dependencies = [
+  'OperatorFilterAssetSubscription_deploy',
   'TRUSTED_FORWARDER_V2',
   'RoyaltyManager_deploy',
   'OperatorFilter_setup',
```

Comment:
Remove `OperatorFilter_setup` dependency?

Reply:
Right, that tag doesn't exist anymore.